dobromir-hristov/swift-docc-render
# Documentation Pages Content Style Changes

Bug/issue #, if applicable: 83860442

## Summary
This PR introduces a few new UI changes on the Documentation pages:
1. Removed right side bar and on this page navigation component
2. Moved Availability to the Hero image
3. Moved Technology information to the ribbon under the Hero image
4. Content Style changes 

This PR covers work proposed in this Forum Post:
https://forums.swift.org/t/new-swift-docc-content-styling/55326

## Dependencies
https://github.com/apple/swift-docc-render/pull/51

## Testing
Make sure you download and unzip the `.doccarchive` - [SlothCreator.doccarchive.zip](https://github.com/apple/swift-docc-render/files/8082216/SlothCreator.doccarchive.1.zip)

Steps:
1. run `VUE_APP_DEV_SERVER_PROXY=/path/to/SlothCreator.doccarchive npm run serve`
2. Visit http://localhost:8080/documentation/slothcreator
3. Assert that the right side bar and on this page navigation component is removed
4. Assert that there is a ribbon beneath the hero image that displays the "technology" a page belongs to
<img width="962" alt="Screen Shot 2022-02-16 at 3 15 51 PM" src="https://user-images.githubusercontent.com/87735557/154373524-46c4c17e-f022-4c06-8282-565cb97e880c.png">

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
